redisinsight/ui/src/constants/cliOutput.tsx

```diff
@@ -92,13 +94,19 @@ export const cliTexts = {
     ]
   ),
   MONITOR_COMMAND: (onClick: () => void) => (
-    <EuiTextColor color="danger" key={Date.now()}>
-      {'Use '}
-      <EuiLink onClick={onClick} className="btnLikeLink" color="text" data-test-subj="monitor-btn">
-        Profiler
-      </EuiLink>
-      {' tool to see all the requests processed by the server.'}
-    </EuiTextColor>
+    <FeatureFlagComponent
+      name={FeatureFlags.disabledByEnv}
+      enabledByDefault
+      otherwise={<EuiTextColor color="accent">Profiler not implemented in this environment.</EuiTextColor>}
```

Comment:
Sorry to be pedantic, but can we change the message to "Monitor not supported in this environment." ?

Mentioning profiler (which is a Redis Insight desktop feature) when trying to run the Redis command Monitor seems misleading. Saying something isn't implemented makes it sound like we haven't gotten around to writing the code, whereas saying it isn't supported leaves it more open ended.